tests/posix_time/main.c

```diff
@@ -23,22 +24,22 @@
 
 int main(void)
 {
-    puts("usleep 1 x 1000*1000");
-    for (int i = 0; i < 10; i++) {
-        useconds_t us = i*1000u*1000u;
-        printf("calling usleep(%u)\n", (unsigned int) us);
+    puts("Please hit any key and then ENTER to continue");
+    getchar();
+    puts("5 x usleep(i++ * 500000)");
+    for (int i = 0; i < 5; i++) {
+        useconds_t us = i*500000u;
         usleep(us);
         puts("wake up");
     }
 
-    puts("sleep 1");
-    for (int i = 0; i < 10; i++) {
+    puts("5 x sleep(i++)");
+    for (int i = 0; i < 5; i++) {
```

Comment:
you could use `unsigned i = 0` here and omit `s` below completely - but I guess the compiler optimises that already anyway.

Reply:
True, will do.